--- src/usr.sbin/rpki-client/validate.c.orig	Fri Sep 11 08:43:39 2026
+++ src/usr.sbin/rpki-client/validate.c	Fri Sep 11 12:39:00 2026
@@ -292,7 +292,7 @@ valid_uri(const char *uri, size_t usz, const char *pro
 	}
 
 	/* do not allow files or directories to start with a '.' */
-	if (strstr(uri, "/.") != NULL)
+	if (memmem(uri, usz, "/.", strlen("/.")) != NULL)
 		return 0;
 
 	if (strncasecmp(uri, RSYNC_PROTO, RSYNC_PROTO_LEN) == 0) {
